cJSON.c

```diff
@@ -2132,6 +2138,21 @@ CJSON_PUBLIC(cJSON*) cJSON_AddNumberToObject(cJSON * const object, const char *
     return NULL;
 }
 
+CJSON_PUBLIC(cJSON*) cJSON_AddNumberToObjectOpts(cJSON * const object, const char * const name, const double n, int places)
```

Comment:
If you are going to ignore values above 14, I think it would be best to fail here rather than silently fall back to the default behaviour.  
Similar for negative values.

Reply:
Yes that's a valid place to do sanity check. I will update the patches once we close other comments 👍